.github/workflows/ci.yml

```diff
@@ -41,11 +46,12 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Install Ruby and gems
         uses: ruby/setup-ruby@v1
         with:
+          ruby-version: 3.2
```

Comment:
This action reads .ruby-version, so setting the version explicitly seems like unnecessary risk of this becoming incorrect?